R/deploy.R

```diff
@@ -487,49 +487,65 @@ has_image <- function(content) {
 #'
 #' Set the Content Image using a variety of methods.
 #'
-#' NOTE: `set_image_webshot()` requires [webshot2::webshot()], but currently
-#' skips and warns for any content that requires authentication until the
-#' `webshot2` package supports authentication.
-#'
 #' @param content A content object
-#' @param path The path to an image on disk
-#' @param url The url for an image
-#' @param ... Additional arguments passed on to [webshot2::webshot()]
+#' @param path A file path or URL to an image
 #'
-#' @rdname set_image
 #' @family content functions
+#' @rdname set_content_image
 #' @export
-set_image_path <- function(content, path) {
-  warn_experimental("set_image_path")
+set_content_image <- function(content, path) {
+  warn_experimental("set_content_image")
```

Comment:
Does this need to be experimental?

Reply:
The package's previous policy seems to have been to use this warning for all unversioned URLs, which this function still uses. I don't feel strongly about keeping the warning; however, changing it opens questions in my mind about all the other places it's used.

I guess, it doesn't seem like the `v1` / unversioned nature of our API actually corresponds to "experimental" / "stable", and definitely not for user-facing functions. And since we're planning to make all unversioned APIs versioned, perhaps all of these warnings can go away?

Reply:
https://github.com/rstudio/connectapi/issues/304 - I neglected to create a connectapi tracking issue previously.